knowledge-base/chat-croll-to-bottom.md

```diff
@@ -3,8 +3,8 @@ title: Scroll KendoReact Conversational UI to the bottom automatically
 description: An example on how to make the Conversational UI scroll to the bottom automatically
 type: how-to
 page_title: Scroll to the bottom of each message automatically - KendoReact Conversational UI
-slug: conversational-ui-scroll-to-bottom-automatically
-tags: conversational-ui, scroll, scroll to bottom, automatically, scroll automatically
+slug: chat-scroll-to-bottom-automatically
```

Comment:
We'd rather avoid updating the slugs of already published resources. Uou can update the Description and the Title. though